README.md

```diff
@@ -79,65 +82,77 @@ The Radeon Augmentation Library - [RALI](rali/README.md) is designed to efficien
-  * Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
+  + Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
 * [FFMPEG n4.0.4](https://github.com/FFmpeg/FFmpeg/releases/tag/n4.0.4) - Optional
-  * FFMPEG is required for amd_media & mv_deploy modules
-  
+  + FFMPEG is required for amd_media & mv_deploy modules
+
 #### Prerequisites setup script for Linux - `MIVisionX-setup.py`
 
 For the convenience of the developer, we here provide the setup script which will install all the dependencies required by this project.
 
 **MIVisionX-setup.py** builds all the prerequisites required by MIVisionX. The setup script creates a deps folder and installs all the prerequisites, this script only needs to be executed once. If the directory option is not given, the script will install the deps folder in the home directory(~/) by default, else in the user-specified location.
 
 ##### Prerequisites for running the script
-1. Ubuntu `16.04`/`18.04` or CentOS `7.5`/`7.6`
-2. X Window
-3. [ROCm supported hardware](https://rocm.github.io/hardware.html)
-4. [ROCm](https://github.com/RadeonOpenCompute/ROCm#installing-from-amd-rocm-repositories)
+
+1. Ubuntu `16.04` / `18.04` or CentOS `7.5` / `7.6`
+2. [ROCm supported hardware](https://rocm.github.io/hardware.html)
+3. [ROCm](https://github.com/RadeonOpenCompute/ROCm#installing-from-amd-rocm-repositories)
 
 **usage:**
-````
-python MIVisionX-setup.py --directory [setup directory - optional]
+
+``` 
+python MIVisionX-setup.py --directory [setup directory - optional (default:~/)]
                           --installer [Package management tool - optional (default:apt-get) [options: Ubuntu:apt-get;CentOS:yum]]
+                          --opencv    [OpenCV Version - optional (default:3.4.0)]
                           --miopen    [MIOpen Version - optional (default:2.5.0)]
                           --miopengemm[MIOpenGEMM Version - optional (default:1.1.5)]
-                          --ffmpeg    [FFMPEG Installation - optional (default:no) [options:Install ffmpeg - yes]]
-                          --rpp       [RPP Installation - optional (default:yes) [options:yes/no]]
+                          --protobuf  [ProtoBuf Version - optional (default:3.12.0)]
+                          --rpp       [RPP Version - optional (default:0.4)]
+                          --ffmpeg    [FFMPEG Installation - optional (default:no) [options:yes/no]]
+                          --rali      [MIVisionX RALI Dependency Install - optional (default:yes) [options:yes/no]]
+                          --neural_net[MIVisionX Neural Net Dependency Install - optional (default:yes) [options:yes/no]]
                           --reinstall [Remove previous setup and reinstall (default:no)[options:yes/no]]
-````
-**Note:** use `--installer yum` for **CentOS**
+```
 
-**Note:** ROCm upgrade with `sudo apt upgrade` requires the setup script rerun. 
+**Note:** 
+
+  + use `--installer yum` for **CentOS**
```

Comment:
Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)